files/perf-fio/fiopod.yaml.in

```diff
@@ -0,0 +1,36 @@
+kind: PersistentVolumeClaim
```

Comment:
You pulled this file into your PR by accident.    This is part of the new fio automation